File: Warmup-1/posNeg/PosNeg.java
package posNeg;

public class PosNeg {
    //Retorne true se um dos números for positivo e o outro negativo, caso o parâmetro "negative" esteja como "true"
    //retorne true se ambos forem negativos.

    public boolean posNeg(int a, int b, boolean negative) {
        if (negative){
            return (a < 0 && b < 0);
        } else return (a * b < 0);
    }
}

File: Python/Warmup-1/pos_neg.py
"""
Retorne true se um dos números dados for negativo e o outro positivo, exceto quando
o parâmetro "negative" for true, nesse caso retorne true quando ambos forem negativos.
"""


def pos_neg(a, b, negative):
    if negative:
        return a < 0 and b < 0
    else:
        return (a < 0 or b < 0) and (a >= 0 or b >= 0)


print(pos_neg(-4, -1, True))
